### PR TITLE
fix comma separation in connectedusernames

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

File: shared/common-adapters/usernames/index.tsx [hunk 1]
```diff
@@ -128,10 +127,14 @@ function UsernameText(props: Props) {
...
-                ,{props.users.length > 2 && i !== props.users.length - 2 ? space : null}
+                ,
+              </Text>
+            )}
+            {i < props.users.length - 1 && (
+              <Text type={props.type} negative={isNegative} style={derivedJoinerStyle}>
+                {space}
               </Text>
             )}
-            {i !== props.users.length - 1 && ' '}
```
Comment (heronhaye) [inline shared/common-adapters/usernames/index.tsx]: don't think this did anything

File: shared/common-adapters/usernames/index.tsx [hunk 2]
```diff
@@ -128,10 +127,14 @@ function UsernameText(props: Props) {
             {/* Injecting the commas here so we never wrap and have newlines starting with a , */}
             {i !== props.users.length - 1 && (!props.inlineGrammar || props.users.length > 2) && (
```
Comment (heronhaye) [inline shared/common-adapters/usernames/index.tsx]: is inlineGrammar the same as showAnd? Can we get rid of inlineGrammar?

Comment (jakob223) [reply shared/common-adapters/usernames/index.tsx]: no idea. who wrote this?

Comment (heronhaye) [reply shared/common-adapters/usernames/index.tsx]: Nojima

**Pull request merged**